packages/techdocs-common/api-report.md

```diff
@@ -91,78 +88,74 @@ export const getDocFilesFromRepository: (
     | undefined,
 ) => Promise<PreparerResponse>;
 
-// Warning: (ae-missing-release-tag) "getLocationForEntity" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
-//
-// @public (undocumented)
+// @public
 export const getLocationForEntity: (
   entity: Entity,
   scmIntegration: ScmIntegrationRegistry,
 ) => ParsedLocationAnnotation;
 
-// Warning: (ae-missing-release-tag) "ParsedLocationAnnotation" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
-//
-// @public (undocumented)
+// @public
+export type MigrateRequest = {
+  removeOriginal?: boolean;
+  concurrency?: number;
+};
+
+// @public
 export type ParsedLocationAnnotation = {
   type: RemoteProtocol;
   target: string;
 };
 
-// Warning: (ae-missing-release-tag) "parseReferenceAnnotation" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
-//
-// @public (undocumented)
+// @public
 export const parseReferenceAnnotation: (
   annotationName: string,
   entity: Entity,
 ) => ParsedLocationAnnotation;
 
-// Warning: (ae-missing-release-tag) "PreparerBase" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
-//
-// @public (undocumented)
+// @public
 export type PreparerBase = {
-  prepare(
-    entity: Entity,
-    options?: {
-      logger?: Logger_2;
-      etag?: string;
-    },
-  ): Promise<PreparerResponse>;
+  prepare(entity: Entity, options?: PreparerOptions): Promise<PreparerResponse>;
 };
 
-// Warning: (ae-missing-release-tag) "PreparerBuilder" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
-//
-// @public (undocumented)
+// @public
 export type PreparerBuilder = {
   register(protocol: RemoteProtocol, preparer: PreparerBase): void;
   get(entity: Entity): PreparerBase;
 };
 
-// Warning: (ae-missing-release-tag) "Preparers" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
-//
-// @public (undocumented)
+// @public
+export type PreparerFactory = {
+  logger: Logger_2;
+  reader: UrlReader;
+};
+
+// @public
+export type PreparerOptions = {
```

Comment:
And _this_ could be `PrepareOptions`?  Or `PreparationOptions`?

Reply:
Or all of the above `*Factory` -> `*Options` suggestions could be `*FactoryOptions` instead.  ...  And this could stay `PreparerOptions`.

I only comment about these because it feels odd for an argument of type factory, rather than the class itself.

Reply:
Renamed to `PreparerConfig`, https://github.com/backstage/backstage/pull/9718/commits/e257a8f0f499672b3e696df53d0604aba9ddbf0b.